GSA/openacr-website
# Refresh openacr table and pa11y ci update

Refreshed the table to match changes from https://github.com/GSA/openacr/issues/280 and also updated pa11y-ci.

**QA**

* Confirm the table changes here: https://federalist-6d3494a6-5b03-42fa-82f2-57f3bbb203af.app.cloud.gov/preview/gsa/openacr-website/refresh-openacr-table-pa11y-ci-update/openacr/
* Confirm the a11y tests pass: https://github.com/GSA/openacr-website/runs/4888886642?check_suite_focus=true
